subsys/net/lib/nrf_cloud/src/nrf_cloud_fota.c

```diff
@@ -104,6 +106,31 @@ static bool reboot_on_init;
  */
 static bool fota_report_ack_pending;
 
+static void update_last_job(char const *const job_id)
+{
+	if ((job_id == NULL) || (job_id[0] == '\0')) {
+		return;
+	}
+
+	K_SPINLOCK(&last_job_slock) {
+		strncpy(last_job, job_id, sizeof(last_job) - 1);
```

Comment:
Maybe we should print a warning if the job ID is too large?

Or do you think that is too unlikely to happen to justify the memory cost?

Reply:
job id should have already been checked for length.